frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-Page-Smoke-Metrics-ODC-4116.feature

```diff
@@ -0,0 +1,165 @@
+Feature: Metrics Tab on Monitoring Page
+    As a user, I should be able to run queries to check the usage
+
+
+Background: 
+    Given user is at Developer perspective
+
+
+@smoke, @regression
+Scenario: CPU Usage Query on Metrics tab: MH-01-TC04
+    Given user is on Monitoring page
+    When user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects CPU Usage Query
+    Then user will see the CPU Usage for project
+    And user will see the pods list inside the project
+    And user will see the value of CPU used by each pod
+
+
+@smoke, @regression
+Scenario: Memory Usage Query on Metrics tab: MH-01-TC04
+    Given user is on Monitoring page
+    When user clicks on Metrics tab
+    And user clicks on Select Query dropdown
+    And user selects Memory Usage Query
```

Comment:
And user selects "Memory usage Query" from Select Query drop down

This will be easily understandable and number of lines gets reduced and also step is re-usable for other scenarios as well.. Helpful for us in automation as well

Reply:
Updated